sandpack-react/src/components/CodeEditor/CodeMirror.tsx

```diff
@@ -125,7 +125,10 @@ export const CodeMirror = React.forwardRef<CodeMirrorRef, CodeMirrorProps>(
 
     const c = useClasser("sp");
     const { listen } = useSandpack();
-    const ariaId = React.useRef<string>(id ?? generateRandomId());
```

Comment:
Note sure if the "passed-in ID" use case is important.

Reply:
Initially, the purpose of the `id` props is to get consistent ids in snapshot tests. But I guess `useId` will always return a constant value, right?

Reply:
As long as you don't change the parent component path. If you do, the snapshot will update.